packages/api/src/sendPayment/sendPayment.ts

```diff
@@ -1,26 +1,28 @@
 import {
   GEM_WALLET,
-  PaymentResponse,
   PaymentRequestPayload,
-  RequestPaymentMessage
+  RequestSendPaymentMessage,
+  SendPaymentResponse
 } from '@gemwallet/constants';
 
 import { sendMessageToContentScript } from '../helpers/extensionMessaging';
 
-export const sendPayment = async (payment: PaymentRequestPayload) => {
+export const sendPayment = async (paymentPayload: PaymentRequestPayload) => {
```

Comment:
```suggestion
export const sendPayment = async (paymentPayload: SendPaymentRequestPayload) => {
```
Please also export `SendPaymentRequestPayload` within the API